PWGHF/HFL/DataModel/ElectronSelectionTable.h

```diff
@@ -49,6 +49,8 @@ DECLARE_SOA_COLUMN(TimeEmcCluster, timeEmcCluster, float);       //! time of the
 DECLARE_SOA_COLUMN(DeltaEtaMatch, deltaEtaMatch, float); //! dEta matched track to EMCal cluster
 DECLARE_SOA_COLUMN(DeltaPhiMatch, deltaPhiMatch, float); //! dPhi matched track to EMCal cluster
 DECLARE_SOA_COLUMN(ISEmcal, isEmcal, bool);              //! electron information with Emcal
+DECLARE_SOA_COLUMN(IsLSElectron, isLSElectron, bool);    //! Like sign electron information with Emcal
```

Comment:
Adding the LS and ULS in the same table as inclusive electron might not be ideal since you could duplicate the inclusive electron information for every ULS and LS pair. It might be good to have a new table with LS and ULS pair information and include maybe a column for inclusive electron ID